kasa/smart/smartdevice.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         self._parent: SmartDevice | None = None
         self._children: Mapping[str, SmartDevice] = {}
         self._last_update = {}
+        self._is_cloud_connected = False
```

Comment:
```suggestion
```
This probably should not exist, and the module should just use the property to access the info.

Reply:
Fixed